Simbody/Visualizer/simbody-visualizer/simbody-visualizer.cpp

```diff
@@ -98,16 +99,22 @@ static void shutdown();
 #ifdef _WIN32
     #include <io.h>
     #define READ _read
+    #define WRITEFUNC _write
 #else
     #include <unistd.h>
     #define READ read
+    #define WRITEFUNC write
+#endif
+
+#ifdef _MSC_VER
+#pragma warning(disable:4996) // don't warn about strerror, sprintf, etc.
 #endif
 
 // gcc 4.4.3 complains bitterly if you don't check the return
 // status from the write() system call. This avoids those
 // warnings and maybe, someday, will catch an error.
 #define WRITE(pipeno, buf, len) \
-   {int status=write((pipeno), (buf), (len)); \
+   {int status=WRITEFUNC((pipeno), (buf), (len)); \
```

Comment:
> What do you think about #undefing this at the end of the file?

Not needed because it isn't a header file. Everything it defines is lost at the end of the compilation unit.